python/test/test_taskqueue.py

```diff
@@ -0,0 +1,79 @@
+import os
+
+import pytest
+
+from neuroglancer.pipeline.tasks import RegisteredTask
+from neuroglancer.pipeline import TaskQueue
+
+TRAVIS_BRANCH = None if 'TRAVIS_BRANCH' not in os.environ else os.environ['TRAVIS_BRANCH']
+
+if TRAVIS_BRANCH is None:
+  QUEUE_NAME = 'test-pull-queue'
+elif TRAVIS_BRANCH == 'master':
```

Comment:
It was that travis/pr and travis/push both ran at the same time and both were adding to and purging from the same queue, which violated assumptions in the tests.